Android/app/src/main/java/com/cs4125/bikerentalapp/repository/BikeRepository.java

```diff
@@ -42,7 +42,7 @@ public BikeRepository(Webservice webservice, Executor executor){
         return liveResponse;
     }
 
-    public LiveData<ResponseBody> placeOrder(String username, int bikeId, int paid){
+    public LiveData<ResponseBody> Rent(int bikeId, int userId){
```

Comment:
function names should begin lowercase

Reply:
same goes for in the webservice (call on line 54 is using uppercase method name as well)